README.md

```diff
@@ -94,12 +92,16 @@ Follow [OperatorHub instructions](https://operatorhub.io/operator/service-bindin
 
 Clone the repository and run `make local` using an existing `kube:admin` kube context.
 
+### Running Tests
+
+* Clone the repository and run `make test-unit` for unit tests.
+* Follow [instructions](test/acceptance/README.md) to run acceptance tests.
```

Comment:
As https://github.com/redhat-developer/service-binding-operator/issues/745 suggest, we should update CONTRIBUTION.md IMHO, README should be kept short and clear, pointing to the other documents providing more details about given topics.

Reply:
I see that `make test`  has been added https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#pull-requests-for-code-and-documentation  . So I would be closing the PR. Please feel free to close #745